src/main/resources/static/app/js/controllers.js

```diff
@@ -0,0 +1,12 @@
+(function () {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Move the invocation into the parens that contain the function.](https://www.codacy.com/app/martzrp/booktrackr/file/2613223914/issues/source?bid=3283479&fileBranchId=3311775#l1)